### PR TITLE
update and info store maps etc

### PR DESCRIPTION
**Pull request by Javitocatral**

### CODE REVIEW

File: src/lib/components/footer.svelte [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
       <h3>Ophavsret &#169; 2021,</h3>
       <h3>Vermouth NU.</h3>
-      <h3>Drevet af Shopify</h3>
+      <h3>CVR - 40649395</h3>
     </div>
   </div>
   <div>
     <div class="right">
       <h3>Newsletter</h3>
       <h3>tlf:784-449-333</h3>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:22]: delete

File: src/lib/components/footer.svelte [hunk 2]
```diff
@@ -13,14 +13,14 @@
...
-      <h3>noseymeil@gmail.com</h3>
+      <h3>info@vermouth.nu</h3>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:23]: mailto link

File: src/lib/components/link.svelte [hunk 3]
```diff
@@ -6,13 +6,12 @@
...
 <style>
   .link {
-    display: flex;
     text-decoration: none;
     font-weight: bold;
     color: var(--interactive-button);
     position: relative;
-    justify-content: space-between;
     z-index: 1;
+    gap: 10px;
```
Comment (zwergius) [inline src/lib/components/link.svelte:14]: gap only works with flex

File: src/routes/list-store/+page.svelte [hunk 4]
```diff
@@ -50,7 +50,7 @@
     <ul>
       {#each stores as { name, href }}
         <li>
-          <Links {href}><span class="truncate">{name}</span> &#8594;</Links>
+          <Links {href}><span class="truncate">{name} </span> &#8594;</Links>
```
Comment (zwergius) [inline src/routes/list-store/+page.svelte:53]: no extra spaces

Comment (zwergius) [reply src/routes/list-store/+page.svelte:53]: not sure why this is plural?

**Review by zwergius — CHANGES_REQUESTED** [review]
image

not filled entirely when selected

File: src/lib/components/link.svelte (hunk 3 above)
Comment (Javitocatral) [reply src/lib/components/link.svelte:14]: It seems that flex is no longer needed because if you separate the content from the link

File: src/lib/components/footer.svelte [hunk 5]
```diff
@@ -19,15 +19,16 @@
   <div>
     <div class="right">
       <h3>Newsletter</h3>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:21]: also not needed

File: src/routes/list-store/+page.svelte [hunk 6]
```diff
@@ -50,7 +50,7 @@
     <ul>
       {#each stores as { name, href }}
         <li>
-          <Links {href}><span class="truncate">{name} </span> &#8594;</Links>
+          <Links {href}><span class="truncate">{name}</span> &#8594;</Links>
```
Comment (zwergius) [inline src/routes/list-store/+page.svelte:53]: Still a space and still plural

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/lib/components/footer.svelte [hunk 7]
```diff
@@ -1,38 +1,39 @@
...
+      <h3>CVR - 40649395</h3>
+    </div>
+  </div>
+  <div>
+    <div class="right">
       <div class="social-media">
         <h3>F&oslash;lg os</h3>
         <a href="https://www.instagram.com">
-          <h3>Ig</h3>
+          <h2>Ig</h2>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:14]: Semantically incorrect, these are not headers

File: src/lib/components/footer.svelte [hunk 8]
```diff
@@ -1,38 +1,39 @@
 <div class="container">
   <div>
     <div>
+      <h3>Ophavsret &#169; 2021,</h3>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:4]: not headers

File: src/lib/components/footer.svelte [hunk 9]
```diff
@@ -1,38 +1,39 @@
...
-      <h3>Drevet af Shopify</h3>
-    </div>
-  </div>
-  <div>
-    <div class="right">
-      <h3>Newsletter</h3>
-      <h3>tlf:784-449-333</h3>
-      <h3>noseymeil@gmail.com</h3>
+      <a href="mailto:info@vermouth.nu">
+        <h3>info@vermouth.nu</h3>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:22]: not headers

File: src/lib/components/link.svelte (hunk 3 above)
Comment (zwergius) [reply src/lib/components/link.svelte:14]: exactly, if it's not needed we delete it

File: src/lib/components/product.svelte [hunk 10]
```diff
@@ -40,8 +40,11 @@
     margin-bottom: 20px;
   }
   @media (hover: hover) {
-    .product:hover {
+    /* .product:hover {
```
Comment (zwergius) [inline src/lib/components/product.svelte:43]: when did we decide to remove this?

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/lib/components/product.svelte (hunk 10 above)
Comment (Javitocatral) [reply src/lib/components/product.svelte:43]: I removed it because I was making an animation that would show when hovering that would show the technical sheet was a test

Comment (zwergius) [reply src/lib/components/product.svelte:43]: But you've kept the cursor pointer - delete that as well then, since it's not clickable...

File: src/routes/list-store/+page.svelte [hunk 11]
```diff
@@ -82,6 +82,9 @@
   ul {
     padding-bottom: 20px;
   }
+  .truncate {
+    margin-right: 10px;
```
Comment (zwergius) [inline src/routes/list-store/+page.svelte:86]: we don't use margin-right as you must remember

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/lib/components/footer.svelte [hunk 12]
```diff
@@ -1,42 +1,44 @@
...
-  <div>
-    <div class="right">
-      <h3>Newsletter</h3>
-      <h3>tlf:784-449-333</h3>
-      <h3>noseymeil@gmail.com</h3>
-      <h3>Nyhedsbrev</h3>
+      <a href="mailto:info@vermouth.nu">
+        <p>info@vermouth.nu</p>
+      </a>
+      <p>Nyhedsbrev</p>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:24]: delete

File: src/lib/components/footer.svelte [hunk 13]
```diff
@@ -1,42 +1,44 @@
...
     </div>
   </div>
 </div>
 
 <style>
-  a h3 {
+  .social-media a p {
```
Comment (zwergius) [inline src/lib/components/footer.svelte:30]: link component should have ```color: inherit``` already, why is this needed ?

File: src/lib/components/footer.svelte [hunk 14]
```diff
@@ -1,42 +1,44 @@
...
-    </div>
-  </div>
-  <div>
-    <div class="right">
-      <h3>Newsletter</h3>
-      <h3>tlf:784-449-333</h3>
-      <h3>noseymeil@gmail.com</h3>
-      <h3>Nyhedsbrev</h3>
+      <a href="mailto:info@vermouth.nu">
+        <p>info@vermouth.nu</p>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:22]: you don't need to have p tag here

File: src/lib/components/footer.svelte [hunk 15]
```diff
@@ -1,42 +1,44 @@
 <div class="container">
   <div>
     <div>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:3]: extra div, can it be removed?

File: src/lib/components/footer.svelte [hunk 16]
```diff
@@ -1,42 +1,44 @@
...
+    </div>
+  </div>
+  <div>
+    <div class="right">
       <div class="social-media">
-        <h3>F&oslash;lg os</h3>
+        <p>F&oslash;lg os</p>
         <a href="https://www.instagram.com">
-          <h3>Ig</h3>
+          <p>Ig</p>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:14]: you can remove p

File: src/lib/components/footer.svelte [hunk 17]
```diff
@@ -1,42 +1,44 @@
...
         </a>
 
         <a href="https://es-es.facebook.com">
-          <h3>Fb</h3>
+          <p>Fb</p>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:18]: you can remove p

File: src/lib/components/link.svelte [hunk 18]
```diff
@@ -27,6 +26,12 @@
     transform-origin: bottom;
     transition: transform 0.2s ease-in-out;
   }
+  @media only screen and (max-width: 1024px) {
```
Comment (zwergius) [inline src/lib/components/link.svelte:29]: why do you add this?

File: src/routes/list-store/+page.svelte [hunk 19]
```diff
@@ -1,67 +1,92 @@
...
     },
     {
-      name: 'Tres cerditos Restaurante Madrid',
-      href: 'https://www.google.es/maps/place/Palos+de+la+Frontera,+Madrid/@40.4039389,-3.6992133,16z/data=!3m1!4b1!4m6!3m5!1s0xd42262fc21d3803:0x91783e9a241dff74!8m2!3d40.4039957!4d-3.6999529!16s%2Fg%2F120nf699?hl=es&entry=ttu',
+      name: 'Vinvetto i Torvehallerne',
+      href: 'https://www.google.es/maps/place/Vinveto/@55.6843847,12.5669551,17.46z/data=!4m6!3m5!1s0x4652533f332b2c8b:0x12de88491b375343!8m2!3d55.6836436!4d12.5695507!16s%2Fg%2F11fht90tlk?entry=ttu',
+    },
+  ]
+
+  const bar = [
```
Comment (zwergius) [inline src/routes/list-store/+page.svelte:19]: plural

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/lib/components/link.svelte (hunk 18 above)
Comment (Javitocatral) [reply src/lib/components/link.svelte:29]: because for tablet it has a different padding

Comment (zwergius) [reply src/lib/components/link.svelte:29]: why ?

Comment (zwergius) [reply src/lib/components/link.svelte:29]: why?

File: src/lib/components/footer.svelte (hunk 13 above)
Comment (Javitocatral) [reply src/lib/components/footer.svelte:30]: I don't know what you mean, the links of the social networks are in red, that's why that style

File: src/lib/components/link.svelte (hunk 18 above)
Comment (Javitocatral) [reply src/lib/components/link.svelte:29]: because if we put the same padding for all the screens it is very focused and it looks really bad but if you want I put everything the same

Comment (zwergius) [reply src/lib/components/link.svelte:29]: if you look at the code it isn't actually changing padding, you're changing the display value - this is unexpected behaviour

File: src/lib/components/footer.svelte [hunk 20]
```diff
@@ -1,49 +1,36 @@
...
-  <div>
-    <div class="right">
-      <h3>Newsletter</h3>
-      <h3>tlf:784-449-333</h3>
-      <h3>noseymeil@gmail.com</h3>
-      <h3>Nyhedsbrev</h3>
+  <div class="right">
+    <div class="social-media">
+      <p>F&oslash;lg os</p>
+      <a href="https://www.instagram.com"> Ig </a>
```
Comment (zwergius) [inline src/lib/components/footer.svelte:10]: ugly spaces

File: src/lib/components/footer.svelte [hunk 21]
```diff
@@ -1,49 +1,36 @@
...
   .social-media {
     gap: 10px;
     display: flex;
-  }
-
-  h3 {
-    font-size: var(--25px);
-    line-height: 1.5;
-    color: var(--secondary-text-color);
+    justify-content: flex-end;
```
Comment (zwergius) [inline src/lib/components/footer.svelte:24]: why is this here

File: src/lib/components/link.svelte [hunk 22]
```diff
@@ -2,15 +2,17 @@
   export let href: string
 </script>
 
-<a {href} class="link"><slot /></a>
+<a class="link" rel="noopener" {href}><slot /></a>
```
Comment (zwergius) [inline src/lib/components/link.svelte:5]: needs external as well, lookup links in SvelteKit docs

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/lib/components/footer.svelte (hunk 21 above)
Comment (Javitocatral) [reply src/lib/components/footer.svelte:24]: because if not social media aligns on the left

Comment (zwergius) [reply src/lib/components/footer.svelte:24]: justify-content ONLY affects flex containers, where is the flex coming from?

**Review by zwergius — APPROVED** [review]
